packages/block-library/src/audio/editor.scss

```diff
@@ -2,4 +2,18 @@
 	// Remove the left and right margin the figure is born with.
 	margin-left: 0;
 	margin-right: 0;
+	position: relative;
+
+	&.is-transient video {
```

Comment:
Why do we style a Video element in the Audio player?

Reply:
Oh, it's copy-pasta 😅  I will push the fix shortly.

Reply:
Fixed in 4eba194.